src/Symfony/Component/Console/composer.json

```diff
@@ -27,6 +27,7 @@
     "suggest": {
         "symfony/event-dispatcher": "",
         "symfony/process": "",
+        "symfony/filesystem": "",
```

Comment:
There's no proper error handling when this feature is used if the filesystem is not present (rare but possible).